internal/test/framework/framework.go

```diff
@@ -194,6 +201,21 @@ func (f *TestFramework) getOpenShiftConfigClient() error {
 	return nil
 }
 
+// getOpenShiftConfigClient gets a new OpenShift operator v1 client
+func (f *TestFramework) getOpenShiftOperatorClient() error {
+	config, err := clientcmd.BuildConfigFromFlags("", kubeconfig)
```

Comment:
Can we try reducing the usage of `kubeconfig` global var and see if we want that var in our framework. I am fine using the `KUBECONFIG` environment var for now.